components/TradingCardGrid.jsx

```diff
@@ -1,10 +1,22 @@
+import { useFilterContext } from '../context/FilterContext';
 import getFullName from '../util/getFullName';
 import TradingCard from './TradingCard';
 
 export default function TradingCardGrid({ teamMembers }) {
+  const { project, roles, availability } = useFilterContext();
+
+  const filteredTeamMembersByAvailability = teamMembers
+    .filter((member) => ((member.weeklyCapacity - member.forecastedHours) > 0) === availability);
+
+  const filteredTeamMembersByRole = filteredTeamMembersByAvailability
+    .filter((member) => roles.includes(member.jobTitle));
+
+  const filteredTeamMembers = filteredTeamMembersByRole
+    .filter((member) => project === 'all' || member.project === project);
+
   return (
     <div className="cmp-trading-card-grid">
-      {teamMembers.map((member) => (
+      {filteredTeamMembers.map((member) => (
```

Comment:
`filteredTeamMembers`, using the default state I've included in `FilterContext.jsx`, includes available team members from all roles across all projects. (Sorry, Merry.)